sources/packages/web/src/types/contracts/Common.ts

```diff
@@ -1,22 +1,27 @@
 import { RouteLocationRaw } from "vue-router";
 
-// todo: use MenuInterface and once MenuModel removed from all, delete the MenuModel
+/**
+ * ! Use MenuItemModel and once MenuModel removed from all,
+ * ! delete the MenuModel
```

Comment:
Nothing major but what are the exclamation marks for ?

Reply:
![image](https://user-images.githubusercontent.com/77353155/190520457-a91ead42-02db-42ec-8ffa-df68dd68746b.png)

Reply:
it's updated. pls take look